dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java

```diff
@@ -118,7 +119,7 @@ public static String formatContent(AlertData alertData) {
             for (Map map : list) {
                 for (Entry<String, Object> entry : (Iterable<Entry<String, Object>>) map.entrySet()) {
                     String key = entry.getKey();
-                    String value = entry.getValue().toString();
+                    String value = Objects.nonNull(entry.getValue()) ? entry.getValue().toString() : null;
```

Comment:
I don't think `entry.getValue()` will get npe. Can you provide the reproduce steps?

Reply:
![image](https://github.com/user-attachments/assets/3f27b5b7-32c9-4bc5-afcc-9516a0f9b1fa)
The json field value is null.

Reply:
Where did this parameter `other` comes from? I can't find it.

Reply:
This parameter is used to verify npe and does not exist. 
#16573 
The sql query results are sent via Feishu, and the field value in the query results is null

Reply:
I see. We should do the data convertion in `org.apache.dolphinscheduler.plugin.task.sql.SqlTask`. This ensures that other alarm types are normal.

Reply:
![image](https://github.com/user-attachments/assets/73c83fb0-ab07-4ba8-8cd7-69bb6ff89fed)
Is the field value in the result processed here empty?

Reply:
Yes.

Reply:
![image](https://github.com/user-attachments/assets/d1da59c5-468d-406c-b780-37305b8bc7bb)
Is the second one better?